controls/radwebcam/features/no-camera-content.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Unavailable Camera Content
+page_title: Unavailable Camera Content
+description: This article shows how to set the no or busy camera content with RadWebCam.
+slug: radwebcam-features-no-camera-content
+tags: nocameracontent,busycameracontent,datatemplate
+published: True
+position: 3
+---
+
+# Unavailable Camera Content
+
+The camera control cannot connect a camera if it is already used by another application. In this case and алсо when there is no camera you can show a custom messages. 
```

Comment:
cannot connect **to** a camera**,** if it is already used ..  .... In this case and when there is no camera, you can show  custom messages.